cypress/support/commands/application.js

```diff
@@ -137,7 +137,14 @@ export const getPageData = ({ service, pageType, variant = 'default', id }) => {
   }
   return cy.request(`${Cypress.env('currentPath')}.json`);
 };
+export const getPageDataFromWindow = () => {
+  cy.window().then(win => {
+    const pageData = win.SIMORGH_DATA;
+    return pageData;
+  });
+};
 
 Cypress.Commands.add('testResponseCodeAndType', testResponseCodeAndType);
 Cypress.Commands.add('testResponseCodeAndTypeRetry', testResponseCodeAndType);
 Cypress.Commands.add('getPageData', getPageData);
+Cypress.Commands.add('getPageDataFromWindow', getPageDataFromWindow);
```

Comment:
Just a note that we might need to be careful of a conflict if Lily's article fetching PR is merged first (but we can figure that out depending on which PR is merged first)